src/main/java/com/amplitude/Amplitude.java

```diff
@@ -136,15 +138,27 @@ public synchronized void flushEvents() {
       List<Event> eventsInTransit = new ArrayList<>(eventsToSend);
       eventsToSend.clear();
       CompletableFuture.supplyAsync(
-          () -> {
-            Response response = httpCall.syncHttpCallWithEventsBuffer(eventsInTransit);
-            Status status = response.status;
-
-            if (Retry.shouldRetryForStatus(status)) {
-              Retry.sendEventsWithRetry(eventsInTransit, response, httpCall);
-            }
-            return null;
-          });
+              () -> {
+                Response response = null;
+                try {
```

Comment:
Nit: indent

Reply:
Will not fix this since it's the result after running google java format.